test/fixtures/python/crypto.invalid.py

```diff
@@ -0,0 +1,259 @@
+## Copyright 2019 VMware, Inc.
+## SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Use a simpler example.  This is way too large.

Reply:
We also shouldn't be including other projects code even though this was the code the bug was reporting with.

Reply:
I agree. I will change that example.